app/src/main/res/values-pt-rBR/strings.xml

```diff
@@ -122,7 +122,7 @@
     <!-- Permission messages -->
     <string name="permission_editing_not_allowed">Edição não permitida</string>
     <string name="permission_deleting_not_allowed">Remoção não permitida</string>
-    <string name="permission_pinning_not_allowed">Fixar não permitido</string>
+    <string name="permission_pinning_not_allowed">Pinagem não permitida</string>
```

Comment:
I think the word "Pinagem" doesn't exist in Portuguese.

Reply:
Well, it's basically in that context yes. Like we used to use pins in the school to put leaves in cork boards.

Reply:
Yeah, I think that's fine. :)

Reply:
:+1: